libqtile/widget/battery.py

```diff
@@ -169,6 +169,7 @@ class Battery(_Battery):
          'Display format'
          ),
         ('hide_threshold', None, 'Hide the text when there is enough energy'),
```

Comment:
It might be good to differentiate these two options in the documentation

Reply:
@flacjacket I thought that adding to doc section (just bellow hide_threshold) is enough. Now, I'm not sure where to exactly put it, because I don't see any specific section in documentation (only one which is generated from the code) related to widgets. Could you please point me/give me some hint?